src/stan/io/stan_csv_reader.hpp

```diff
@@ -256,14 +256,14 @@ namespace stan {
             break;
 
           if (comment_line) {
-            if (line.find("(Warm-up)") != std::string::npos) {
-              int left = 17;
-              int right = line.find(" seconds");
+            if (line.find("Warmup") != std::string::npos) {
```

Comment:
What in the world is there a hard coded 17 in here for?  Isn't
that going to make things rather brittle?

> On Aug 19, 2015, at 9:33 AM, Daniel Lee <notifications@github.com> wrote:
> 
> In src/stan/io/stan_csv_reader.hpp:
> 
> > @@ -256,14 +256,14 @@ namespace stan {
> >              break;
> >  
> >            if (comment_line) {
> > -            if (line.find("(Warm-up)") != std::string::npos) {
> > -              int left = 17;
> > -              int right = line.find(" seconds");
> > +            if (line.find("Warmup") != std::string::npos) {
> 
> I'd still like to see the change in behavior as a separate pull request.
> 
> —
> Reply to this email directly or view it on GitHub.
>